README.md

```diff
@@ -467,6 +489,11 @@ users:
     # By default responses aren't cached.
     cache: "longterm"
 
+    # ParamGroup name to use
```

Comment:
Substitute this with:

    An optional group of params to send to ClickHouse with each proxied request.
    These params may be set in param_groups block.

    By default no additional params are sent to ClickHouse.